capstone-backend/app.js

```diff
@@ -115,15 +134,29 @@ app.post ('/profile/interests', async (req, res) => {
   try{
     const Skills = Parse.Object.extend("Skills");
     const skills = new Skills();
+    const skillsData = fs.readFileSync('data/skills.json');
```

Comment:
In nodejs we usually avoid "sync" operations. This can be rewritten with await using fs.promises.
https://stackoverflow.com/questions/46867517/how-to-read-file-with-async-await-properly